core/tests/protractor/embedding.js

```diff
@@ -161,33 +224,46 @@ describe('Embedding', function() {
       driver.get(
         general.SERVER_URL_PREFIX + general.SCRIPTS_URL_SLICE +
         'embedding_tests_dev_i18n_0.0.1.html');
-      browser.switchTo().frame(driver.findElement(by.xpath(
-          "//div[@class='protractor-test-embedded-exploration']/iframe")));
+
+      driver.findElement(
+        by.css('#embedding-exploration-id')).sendKeys(explorationId);
+      driver.findElement(
+        by.css('#embedding-submit-btn')).click();
+
+      browser.switchTo().frame(driver.findElement(
+        by.css('.protractor-test-embedded-exploration iframe')));
       general.waitForSystem();
       browser.waitForAngular();
       expect(driver.findElement(by.css('.protractor-test-float-form-input'))
         .getAttribute('placeholder')).toBe(expectedPlaceholder);
       browser.switchTo().defaultContent();
     };
 
-    users.createUser('embedder2@example.com', 'Embedder2');
+    users.createModerator('embedder2@example.com', 'Embedder2');
     users.login('embedder2@example.com', true);
-    adminPage.reloadExploration('protractor_test_1.yaml');
 
     // Change language to Thai, which is not a supported site language.
-    general.openEditor('12');
+    general.openEditor(explorationId);
     editor.setLanguage('ภาษาไทย');
     editor.saveChanges('Changing the language to a not supported one.');
     // We expect the default language, English
     checkPlaceholder('Type a number');
 
     // Change language to Spanish, which is a supported site language.
-    general.openEditor('12');
+    general.openEditor(explorationId);
     editor.setLanguage('español');
     editor.saveChanges('Changing the language to a supported one.');
     checkPlaceholder('Ingresa un número');
 
     users.logout();
-    general.checkForConsoleErrors([]);
+
+    // This error is to be ignored as 'idToBeReplaced' is not a valid
+    // exploration id. It appers just after the page loads.
+    var errorsToIgnore = new RegExp('http:\/\/localhost:9001\/assets\/' +
```

Comment:
errorToIgnore (singular).

Don't we usually pass strings to checkForConsoleErrors, rather than RegExp objects?

Reply:
@seanlip there is 'version=0.0.1&secret=' where secret will change every time, so i had to use regex.

Reply:
I think if you pass in strings they are automatically regex-matched. Try it and see. I believe we do that elsewhere.

It's fine to go with the following string:

    'http:\/\/localhost:9001\/assets\/scripts\/embedding_tests_dev_i18n_0.0.1.html - Refused to display \'http:\/\/localhost:9001\/explore\/idToBeReplaced\\?iframed=true&locale=en#version=0.0.1&secret='

Reply:
yes it works!! thanks. i have replaced it.